databox/indexer/src/handlers/phraseanet/phraseanetClient.ts

```diff
@@ -105,53 +106,92 @@ export default class PhraseanetClient {
             params,
             offset,
             PhraseanetSearchType.Story,
-            searchQuery
+            searchQuery,
+            20
```

Comment:
Why only 20?

Reply:
arbitrary: global exec time was faster than 50. But since now we don't get story children here anymore (causing huge json), we could try again with 50 or 100